deploy/charts/kubernetes-cluster-prep/templates/clusterrole.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.authnK8s.rbac.create }}
```

Comment:
I think it makes sense to get rid of the `rbac` and use these settings in `values.yaml`, so that we're consistent with the other settings in `values.yaml`:
```
authnK8s:
  clusterRole:
    create: true
    name:
```
If you agree, then this line would be:
```
{{- if .Values.authnK8s.clusterRole.create }}
```

Reply:
yes, I like that better.